libs/novui/src/recipes/sidebar.recipe.ts

```diff
@@ -0,0 +1,23 @@
+import { defineSlotRecipe } from '@pandacss/dev';
+
+export const SIDEBAR_RECIPE = defineSlotRecipe({
+  className: 'sidebar',
+  jsx: ['Sidebar'],
+  slots: ['sidebar', 'header'],
+  base: {
+    sidebar: {
+      padding: 150,
+      backgroundColor: 'sidebar.background',
+      borderRadius: 150,
+      position: 'absolute',
+      right: 50,
+      bottom: 50,
+      top: 50,
+      width: '45%',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** This should be derived from a token. However, in general I think we will want this to be more specifically sized (like in units not percent)